src/pudl/helpers.py

```diff
@@ -1589,7 +1590,10 @@ def post_process(self, value: str) -> str:
         """
         if value is None:
             try:
-                return os.environ[self.env_var]
+                value = os.environ.get(self.env_var)
+
+                if not value:
+                    value = get_defaults()[self.env_var]
             except KeyError:
```

Comment:
I guess this KeyError isn't relevant anymore? Should we keep it around for when we deprecate pudl_settings?